helm/blueapi/values.yaml

```diff
@@ -108,12 +108,13 @@ worker:
       passcode: guest
     host: rabbitmq
     port: 61613
-  # Uncomment this to enable the scratch directory
-  # scratch:
-  #  root: /blueapi-plugins/scratch
-  #   repositories: []
-  #     - name: "dodal"
-  #       remote_url: https://github.com/DiamondLightSource/dodal.git
+
+initContainer:
+  scratch:
+   root: /blueapi-plugins/scratch
+    repositories: []
+      - name: "dodal"
+        remote_url: https://github.com/DiamondLightSource/dodal.git
```

Comment:
This yaml is invalid for both whitespace and setting a value twice. repositories should be `[]` with a commented out example for dodal. Does an auto format touch this file?